Source/Fuse.Elements/Caching/ElementAtlas.uno

```diff
@@ -167,8 +159,12 @@ namespace Fuse.Elements
 			_framebuffer.Unpin();
 		}
 
-		void FillFramebuffer(DrawContext dc, bool drawAll, Rect scissorRectInClipSpace)
+		void FillFramebuffer(DrawContext dc, framebuffer fb, bool drawAll, Rect scissorRectInClipSpace)
 		{
+			// Changing the framebuffer is expensive. This loop actually doesn't draw anything in
+			// common case. Don't push the render target unless we actually have to render something.
```

Comment:
Ah, I think I see why. There's elements that are invalid, but are off-screen. So the cache tries to avoid updating them until it has to... That makes the early-out mechanism not work as it should.